dist/tools/codespell/check.sh

```diff
@@ -37,8 +37,9 @@ ${CODESPELL_CMD} --version &> /dev/null || {
 
 CODESPELL_OPTS="-q 2"  # Disable "WARNING: Binary file"
 CODESPELL_OPTS+=" --check-hidden"
-# Disable false positives "nd  => and, 2nd", "WAN => WANT", "od => of"
-CODESPELL_OPTS+=" --ignore-words-list ND,nd,wan,od,dout"
+# Disable false positives "nd  => and, 2nd", "WAN => WANT", "od => of",
+# "dout => doubt"
+CODESPELL_OPTS+=" --ignore-words-list=ND,nd,wan,od,dout"
```

Comment:
Is `=` addition really needed ?

Reply:
it was there before. I removed it while figuring out why WAN wasn't matching and re-addid it with the fixup.

Reply:
Ah ok :)